src/actions/tests/expressions.spec.js

```diff
@@ -75,13 +76,13 @@ describe("expressions", () => {
     const { dispatch, getState } = createStore(mockThreadClient);
     await createFrames(dispatch);
 
-    dispatch(actions.addExpression("foo"));
-    dispatch(actions.addExpression("bar"));
+    await dispatch(actions.addExpression("foo"));
+    await dispatch(actions.addExpression("bar"));
 
-    expect(selectors.getExpression(getState(), "foo").value).toBe(null);
-    expect(selectors.getExpression(getState(), "bar").value).toBe(null);
+    expect(selectors.getExpression(getState(), "foo").value).toBe("boo");
+    expect(selectors.getExpression(getState(), "bar").value).toBe("boo");
 
-    await dispatch(actions.evaluateExpressions("boo"));
+    await dispatch(actions.evaluateExpressions());
```

Comment:
should we test adding an expression / updating etc w/ errors?

Reply:
I like that idea. Adding a test for error would be great.

Reply:
Tests added.